mobile/macos/Runner/AppDelegate.swift

```diff
@@ -4,7 +4,8 @@ import FlutterMacOS
 @NSApplicationMain
 class AppDelegate: FlutterAppDelegate {
   override func applicationShouldTerminateAfterLastWindowClosed(_ sender: NSApplication) -> Bool {
-    dummy_method_to_enforce_bundling()
+    let dummy = dummy_method_to_enforce_bundling()
+    print(dummy)
```

Comment:
wait, what? 😅 this is fixing the issue 😂

Reply:
yup, dummy methods can be optimised away and therefore stripped in builds if they're not doing anything :-)

Reply:
I just copied the solution referenced in the linked issue

Reply:
leaving this for reference why this fixes the issue: https://github.com/fzyzcjy/flutter_rust_bridge/issues/496#issuecomment-1150038776